utilities/tools/Test-NamePrefixAvailability.ps1

```diff
@@ -86,7 +86,7 @@ function Test-NamePrefixAvailability {
 
         try {
             # Invoke Token Replacement Functionality and Convert Tokens in Parameter Files
-            $parameterFiles | ForEach-Object { $null = Convert-TokensInFile @ConvertTokensInputs -FilePath $_ }
+            $null = $parameterFiles | Convert-TokensInFileList @ConvertTokensInputs
```

Comment:
This should do the trick now, right?
```suggestion
            $null = Convert-TokensInFileList @ConvertTokensInputs -FilePathList $parameterFiles 
```

Reply:
I don't have a strong opinion on which of the two use (pipe or explicit parameter). I'll update this and all other occurrences below to use an explicit parameter as suggested, but will add that to the main `@ConvertTokensInputs`

Reply:
done